js/refresh.js

```diff
@@ -0,0 +1,5 @@
+var user = parameters['user'];
```

Comment:
It is from extractUrlParameters.js . This is the first .js file that is loaded in index.html and plan.html .